components/booking/AvailableTimes.tsx

```diff
@@ -0,0 +1,87 @@
+import dayjs, {Dayjs} from "dayjs";
+import isBetween from 'dayjs/plugin/isBetween';
+dayjs.extend(isBetween);
+import {useEffect, useMemo, useState} from "react";
+import getSlots from "../../lib/slots";
+import Link from "next/link";
+import {timeZone} from "../../lib/clock";
+import {useRouter} from "next/router";
+
+const AvailableTimes = (props) => {
```

Comment:
we should at some point do 

`const AvailableTimes = ({eventType, date, etc.} : {eventType: EventType, date: DateType, etc.}) => {
...
}`

instead of just `props` without types

Reply:
Would the way to do this be a ./lib/types folder?